src/components/ActionBar/src/ActionBarLayer.vue

```diff
@@ -113,6 +113,7 @@ export default {
 
 .Action {
 	margin-right: 8px;
+	-webkit-transform: translate3d(0, 0, 0);
```

Comment:
Can you please add a comment above this style that explains its purpose? As-is it looks like it does nothing and is very likely to get removed by another engineer if there's no comment.